Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:8
+
+WORKDIR /contracts
+
+COPY package.json .
+COPY package-lock.json .
+RUN npm install
+
+COPY ./deploy/package.json ./deploy/
+COPY ./deploy/package-lock.json ./deploy/
+RUN npm install deploy
```

Comment:
Just in case: this is _not_ installing the dependencies inside the `deploy` directory; this is installing a package called [`deploy`](https://www.npmjs.com/package/deploy).